src/document.c

```diff
@@ -446,7 +447,7 @@ FIELD_PREPROCESSOR(fulltextPreprocessor) {
 
       Token tok = {0};
       while (0 != aCtx->tokenizer->Next(aCtx->tokenizer, &tok)) {
-        if ((!strcmp(tok.tok, "")) && !FieldSpec_IndexesEmpty(fs)) {
+        if (strlen(tok.tok) == 0 && !indexesEmpty) {
```

Comment:
```suggestion
        if (!indexesEmpty && strlen(tok.tok) == 0) {
```

Reply:
Assuming that there will be more cases that don't index empty than do (which seems logic at this point).. I can go with this.